internal/export/config.go

```diff
@@ -64,3 +64,96 @@ func (c *Config) KeyManagerConfig() *signing.Config {
 func (c *Config) SecretManagerConfig() *secrets.Config {
 	return &c.SecretManager
 }
+
+// TestConfigDefaults returns a configuration populated with the default values.
```

Comment:
general comment: should the responsibility of providing test config live in the config_test.go files? Or if they need to live together with the definition of the config, is there a way to reduce some of the boilerplate a bit? Or maybe use conditionals to represent the test scenarios. E.g., 
```
CreateTimeout: Env = test ? 1 * time.Minute : 5 * time.Minute
```

Reply:
These exist for two reasons:

1. A failsafe in case defaults are changed
1. A way to share configs with subpackages (e.g. database)

They can't live in `_test.go` because then we can't share them with other packages. For example, anything that depends on the database has database defaults and database overrides. Instead of duplicating that logic everywhere, I chose to export these on the main package. This _is_ reducing the boilerplate 😄

Reply:
We've also been bitten by enough configuration issues that I'm okay with a bunch of code honestly.